src/elements/a-event/index.js

```diff
@@ -1,10 +0,0 @@
-module.exports = {
-  'a-click': require('./a-click'),
-  'a-event': require('./a-event'),
-  'a-hover': require('./a-hover'),
-  'a-load': require('./a-load'),
-  'a-mousedown': require('./a-mousedown'),
```

Comment:
I was going to start recommending `<a-event name="click"...>` instead since that can support every event possible. We can put it in the `CHANGELOG` as a breaking change, and I think it's somewhat okay to break as it was undocumented. I will update examples to all use `<a-event>`.